lib/ronin/fuzzing/core_ext/string.rb

```diff
@@ -161,7 +159,7 @@ def repeating(lengths,&block)
   #     p str
   #   end
   #
-  # @example Replace a `Regexp::UNIX_PATH` with {Ronin::Fuzzing#format_strings}:
+  # @example Replace a [`Regexp::UNIX_PATH`](https://ronin-rb.dev/docs/ronin-support/Regexp.html) (from [ronin-support](https://github.com/ronin-rb/ronin-support) with {Ronin::Fuzzing#format_strings}:
```

Comment:
I think it's redundant to say `from ronin-support` if we're already linking to it.

Reply:
Maybe I can just remove the ronin-support link but keep the text? The idea is that we also can know it comes from another gem just by reading (without clicking on the link).

Reply:
I actually would change the whole line to just `"Replace a UNIX path with every {Ronin::Fuzzing#format_strings}". In the `@param` tag it should explain how Symbols are resolved to `Ronin::Support::Text::Patterns` constants and link to https://ronin-rb.dev/docs/ronin-support/Ronin/Support/Text/Patterns.html.